testcharms/charm-repo/quantal/mysql/icon.svg

```diff
@@ -0,0 +1,335 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Can't we use an in-memory test charm rather than adding to the global charm repo charms?

Also, does the icon have to be the *actual* icon SVG - just a token amount of text would probably do.